pkg/restapi/operation/operations.go

```diff
@@ -24,7 +25,8 @@ import (
 )
 
 const (
-	logModuleName = "restapi"
+	logModuleName                   = "restapi"
+	dataVaultConfigurationVaultName = "data_vault_configurations"
```

Comment:
This isn't a vault - it's just a`store`. Note that a "vault" is something specific in the spec.